### ISSUE TITLE
nice start

### ISSUE THREAD

**Issue by jjhesk** [SudarAbisheck/ZXing-Orient#1]
is this possible to import my own layout xml to make it a part of the scan..

**Comment by b1zantine** [comment]
Its not possible yet. But I am sure it will be available in the near future.

**Comment by RyanNolan** [comment]
Works nicely! Is it possible to increase the size/shape of the "target area"? At present, it is a horizontal rectangle, but when scanning a QR Code, I'd like it to become a Square shape if possible?

**Comment by richardortiz84** [comment]
I agree the small horizontal box when in portrait orientation makes the layout a little less desirable.

**Comment by b1zantine** [comment]
Okay. I will work on it. :+1:
